Dockerfile

```diff
@@ -1,24 +1,28 @@
 # https://github.com/hyrise/hyrise/wiki/Docker-Image
 
-FROM ubuntu:19.04
+FROM ubuntu:19.10
 ENV DEBIAN_FRONTEND noninteractive
 RUN apt-get update \
+    && apt-get install -y software-properties-common \
+    && add-apt-repository -y ppa:mhier/libboost-latest \
```

Comment:
Ubuntu is reeeeally bad when it comes to updating boost. The offered version 1.67 is from April 2018.

Reply:
Can we remove the fix and additional repos when the new server is merged?

Reply:
Not until Ubuntu decides to ship a less antique boost version.

Reply:
... but now that I had my experiences with the code quality of boost, I can somewhat see why they might be hesitant to do so.